MAUI/DigitalGauge/migration.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI SfDigitalGauge | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin SfDigitalGauge to Syncfusion .NET MAUI SfDigitalGauge control and more.
+platform: MAUI
+control: SfDigitalGauge
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfDigitalGauge to .NET MAUI SfDigitalGauge
+
+To make the migration from the Xamarin `SfDigitalGauge` to the .NET MAUI `SfDigitalGauge` easier, most of the similar APIs from the Xamarin `SfDigitalGauge` were kept in the.NET MAUI `SfDigitalGauge`. Also, the APIs has been restructured by considering various use cases and maintaining API consistency. Please find the difference in the following topics.
+
+## Initialize control
+
+To initialize the control, import the gauge namespace and initialize SfDigitalGauge as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfDigitalGauge</th>
+<th>.NET MAUI SfDigitalGauge</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:gauge="clr-namespace:Syncfusion.SfGauge.XForms;assembly=Syncfusion.SfGauge.XForms">
+
+        <gauge:SfDigitalGauge/>
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.SfGauge.XForms;
+...
+
+SfDigitalGauge digitalGauge = new SfDigitalGauge();
+this.Content = digitalGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:gauge="clr-namespace:Syncfusion.Maui.Gauges;assembly=Syncfusion.Maui.Gauges">
+
+        <gauge:SfDigitalGauge/>
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.Gauges;
+...
+
+SfDigitalGauge digitalGauge = new SfDigitalGauge();
+this.Content = digitalGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Character Segments
```

Comment:
## Properties

The following code example, explains how to initialize the properties of the `Xamarin SfDigitalGauge` and `.NET MAUI SfDigitalGauge.`